diff --git a/xlators/mount/fuse/utils/mount.glusterfs.in b/xlators/mount/fuse/utils/mount.glusterfs.in
index 485b466..5ff7968 100755
--- a/xlators/mount/fuse/utils/mount.glusterfs.in
+++ b/xlators/mount/fuse/utils/mount.glusterfs.in
@@ -121,6 +121,12 @@ main ()
     # $2=$(echo "$@" | sed -n 's/[^ ]* \([^ ]*\).*/\1/p');
     
     mount_point="$2";
+
+    # Simple check to avoid multiple identical mounts
+    if grep -q "glusterfs $mount_point fuse" /etc/mtab; then
+        echo "$0: according to mtab, a glusterfs is already mounted on $mount_point"
+        exit 1
+    fi
     
     fs_options=$(echo "$fs_options,$new_fs_options");
     
